segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/ThrottlerCalculator.java

```diff
@@ -81,6 +82,10 @@
      */
     @VisibleForTesting
```

Comment:
please remove this and other constants in code
```
static final int OPERATION_LOG_MAX_SIZE = 1_000_000;
static final int MAX_DELAY_MILLIS = 25000; 
```
in short whatever is coming from config should not be constant.

I suggest move all magic constants to config.

Reply:
All these constants are moved to `ThrottlerPolicy` class. Now, the `ThrottlerClaculator` just performs calculations and the configuration part is encapsulated in `ThrottlerPolicy`.